<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chatting_room_private</class>
 <widget class="QWidget" name="chatting_room_private">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="MessageRecFrom">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>461</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="MessageSendTo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>461</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SendBtn">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>390</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Send(enter)</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BackBtn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>390</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Back(ctrl+B)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </widget>
  <widget class="QLabel" name="User1">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>30</y>
     <width>191</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="User2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>250</y>
     <width>191</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="User2_name">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>230</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>User2</string>
   </property>
  </widget>
  <widget class="QLabel" name="User1_name">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>User1</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
